CheckerUI_xforms/CheckerUI/CheckerUI/CheckerUI/Helpers/Order/OrderItemView.cs

```diff
@@ -153,16 +182,15 @@ public void GenerateGrid()
                 m_Grid.Children.Add(label, 0, 3, i, i+1);
                 i++;
             }
-            m_Grid.Children.Add(m_DoneButton, 0, 1, 4, 5);
-            m_Grid.Children.Add(m_StartButton, 1, 2, 4, 5);
-            m_Grid.Children.Add(m_StopButton, 2, 3, 4, 5);
+
+            addGridButtons();
         }
 
         private void CreateGrid()
         {
             m_Grid = new Grid
             {
-                Margin = new Thickness(0, 5, 0, 5),
+                Margin = new Thickness(-1, 0, 0, 0),
```

Comment:
This grid can be a XAML base for all these definitions.  No point in this being defined in code.

Reply:
Apparently this whole class will be deleted in the next update because the grid is no longer needed